metrics/init.lua

```diff
@@ -93,7 +93,8 @@ return {
     invoke_callbacks = invoke_callbacks,
     set_global_labels = set_global_labels,
     enable_default_metrics = function()
-        return require('metrics.default_metrics.tarantool').enable()
+        require('metrics.default_metrics.luajit').enable()
```

Comment:
```suggestion
        require('metrics.tarantool.luajit').enable()
```

Reply:
and move luajit files

Reply:
Maybe `metrics.default_metrics.tarantool.luajit`?

Reply:
Moved to `metrics.default_metrics.tarantool.luajit`

Reply:
I moved file to `metrics/tarantool/luajit.lua`